api/app/views/mno_enterprise/jpi/v1/marketplace/_app.json.jbuilder

```diff
@@ -8,6 +8,8 @@ json.is_star_ready app.star_ready?
 json.is_connec_ready app.connec_ready?
 json.is_coming_soon app.coming_soon?
 json.single_billing app.single_billing?
+json.multi_instantiable app.multi_instantiable
+json.subcategories app.subcategories
```

Comment:
Do we need the subcategories here?
If not it's better to keep the response smaller.